### ISSUE TITLE
What happens when a template has both f-if and f-each?

### ISSUE THREAD

**Issue by madx** [fluorjs/fluor#12]
Probably a mess as of 0.4.0!

Either we have `f-if` take precedence over `f-each` and that would cancel the iteration, or we forbid to have both tags and the user must use a separate `<template>` tag for each of them:

```html
<template f-if="shown">
  <template f-each="value in values">
```

**Comment by madx** [comment]
I added an example about this: https://github.com/fluorjs/fluor/blob/master/examples.html#L165-L171

**Comment by madx** [comment]
I think the code should remain simple for now and that we should not enforce it on the lib side, let's trust the user. This might however be addressed within #21
